docs/aurora/workflows/libensemble.md

```diff
@@ -1 +1,108 @@
 # libEnsemble on Aurora
+
+[libEnsemble](https://libensemble.readthedocs.io/en/main/) is a Python toolkit for running dynamic ensembles of calculations.
+
+Users provide generator and simulator functions to express their ensembles, where the generator can steer the ensemble based on previous results. These functions can portably submit user executables at any scale.
+
+System details are detected, and dynamic resource management is provided. This includes automatically detecting, assigning, and reassigning GPUs for ensemble members.
+
+libEnsemble can be used in a consistent manner on laptops, clusters, and supercomputers with minimal required dependencies.
+
+## Configuring Python and Installation
+
+To obtain Python use:
+
+    module use /soft/modulefiles
+    module load frameworks
+
+To obtain libEnsemble::
+
+    pip install libensemble
```

Comment:
this will default to a `--user` install, since the base environment site-packages is not user-writable. We really discourage user site-packages installs. 

Can you add the `venv` lines from the https://docs.alcf.anl.gov/aurora/data-science/python/ page?

Reply:
see https://docs.alcf.anl.gov/aurora/data-science/frameworks/pyg/#installing-torch_geometric-on-aurora for example